lib_player_heads/data/gm4_player_heads-1.1/functions/main.mcfunction

```diff
@@ -1,4 +1,4 @@
 schedule function gm4_player_heads-1.1:main 16t
 
 # Repair any dropped player head items.
-execute as @e[type=item,tag=!gm4_player_heads_scanned,nbt={Item:{id:"minecraft:player_head"}},nbt=!{Item:{tag:{gm4_player_head:1b}}}] run function gm4_player_heads-1.1:repair
+execute as @e[type=item,tag=!gm4_player_heads_scanned,nbt={Item:{id:"minecraft:player_head"}},nbt=!{Item:{tag:{display:{}}}}] run function gm4_player_heads-1.1:repair
```

Comment:
I just realized this is a bit flawed. Items that don't match this check will never get the `gm4_player_heads_scanned` tag, and thus will keep checking the nbt conditions. I'm not too concerned about this but it is a place for future optimization.

Reply:
Yeah I realized that flaw too, but it was like this before my changes and I didn't know a super clean way to fix it. I just wanted to getnthe main part of the PR done.